packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
```

Comment:
You should use action-icon as we don't have any camelCase syntax elsewhere in the framework